axelor-project/src/main/java/com/axelor/apps/project/service/roadmap/SprintGeneratorServiceImpl.java

```diff
@@ -164,6 +168,29 @@ public Set<Sprint> generateSprints(
     return sprintSet;
   }
 
+  @Override
+  public String getSprintDomain(Project project) {
+    List<Sprint> sprintList = new ArrayList<>();
+    if (Objects.equals(
+        project.getSprintManagementSelect(), ProjectRepository.SPRINT_MANAGEMENT_PROJECT)) {
+      sprintList = project.getSprintList();
+    }
+    if (Objects.equals(
+        project.getSprintManagementSelect(), ProjectRepository.SPRINT_MANAGEMENT_VERSION)) {
+      List<List<Sprint>> sprintListList =
+          project.getRoadmapSet().stream()
+              .map(ProjectVersion::getSprintList)
+              .collect(Collectors.toList());
+      for (List<Sprint> sprints : sprintListList) sprintList.addAll(sprints);
```

Comment:
You can use flatMap here: https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#flatMap-java.util.function.Function-

```suggestion
      sprintList =
          project.getRoadmapSet().stream()
              .map(ProjectVersion::getSprintList)
              .flatMap(Collection::stream)
              .collect(Collectors.toList());
```